src/common/k8s-api/__tests__/kube-api.test.ts

```diff
@@ -212,9 +212,9 @@ describe("KubeApi", () => {
       await flushPromises();
     });
 
-    it("requests resources from the versioned api group from the initial apiBase", () => {
+    it("requests version list from the api group from the initial apiBase", () => {
```

Comment:
This test file is like 2000 lines long, and covers different technicalities. Split to more files?

Reply:
sure